Update docs (#11226)
DataDog/integrations-core: docs/developer/process/agent-release/pre-release.md

git push origin <MAJOR>.<MINOR>.0-rc.1
```

> We should tag the new release candidate everytime even if there are no new changes for `integrations-core` in the upcoming rc.

Reviewer: ```suggestion
We should create a tag for each new RC, regardless of whether there are new changes from `integrations-core` in the RC.
```
